### PR TITLE
translate api networking resource

### LINKED ISSUE #483: Translate Karmada API document into Chinese
**What would you like to be added**:

We have just added the Karmada API document #476, but the content is in English. Because we currently support Simplified Chinese and English, we need to translate the API document into Chinese.

At present, we do not have an automatic translation tool script to complete this tool. Therefore, drawing on the working method of Kubernetes webstite, we will present the documents to be translated in the form of a list. Anyone who is interested in participating in the translation work can receive the documents they are interested in. Partially, thanks.

- [x] Karmada API @XiShanYongYe-Chang  #487
- [x] Auto Scaling Resources @XiShanYongYe-Chang #497 
  - [x] CronFederatedHPA v1alpha1
  - [x] FederatedHPA v1alpha1
- [x] Cluster Resources @XiShanYongYe-Chang #498 
  - [x] Cluster v1alpha1
- [x] Common Definitions @XiShanYongYe-Chang #503 
  - [x] DeleteOptions
  - [x] LabelSelector
  - [x] ListMeta
  - [x] NodeSelectorRequirement
  - [x] ObjectMeta
  - [x] Patch
  - [x] Quantity
  - [x] Status
  - [x] TypedLocalObjectReference
- [x] Common Parameters @XiShanYongYe-Chang #504
  - [x] Common Parameters
- [x] Config Resources @XiShanYongYe-Chang #499 
  - [x] ResourceInterpreterCustomization v1alpha1
  - [x] ResourceInterpreterWebhookConfiguration v1alpha1
- [ ] Networking Resources @XiShanYongYe-Chang #502 
  - [ ] MultiClusterIngress v1alpha1
  - [ ] MultiClusterService v1alpha1
- [x] Policy Resources @XiShanYongYe-Chang #493 
  - [x] ClusterOverridePolicy v1alpha1
  - [x] ClusterPropagationPolicy v1alpha1
  - [x] FederatedResourceQuota v1alpha1
  - [x] OverridePolicy v1alpha1
  - [x] PropagationPolicy v1alpha1
- [x] Search Resources @XiShanYongYe-Chang #500
  - [x] ResourceRegistry v1alpha1
- [x] Work Resources @XiShanYongYe-Chang #492 
  - [x] ClusterResourceBinding v1alpha2
  - [x] ResourceBinding v1alpha2 
  - [x] Work v1alpha1

/help

You can reply to the content you want to do in the comments, I will assign the corresponding task to you, thanks~

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Translate API networking resource

**Which issue(s) this PR fixes**:
Fixes #483

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/networking-resources/multi-cluster-ingress-v1alpha1.md [hunk 1]
```diff
@@ -32,241 +32,239 @@ MultiClusterIngress is a collection of rules that allow inbound connections to r
 
 - **spec** (IngressSpec)
 
-  Spec is the desired state of the MultiClusterIngress.
+  Spec 是 MultiClusterIngress 所需的状态。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/networking-resources/multi-cluster-ingress-v1alpha1.md:35]:
```suggestion
  Spec 是 MultiClusterIngress 的期望状态。
```
Or
```suggestion
  Spec 是 MultiClusterIngress 的预期状态。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/networking-resources/multi-cluster-service-v1alpha1.md [hunk 2]
```diff
@@ -30,153 +30,155 @@ MultiClusterService is a named abstraction of multi-cluster software service. Th
 
 - **metadata** ([ObjectMeta](../common-definitions/object-meta#objectmeta))
 
-- **spec** ([MultiClusterServiceSpec](../networking-resources/multi-cluster-service-v1alpha1#multiclusterservicespec)), required
+- **spec** ([MultiClusterServiceSpec](../networking-resources/multi-cluster-service-v1alpha1#multiclusterservicespec))，必选
 
-  Spec is the desired state of the MultiClusterService.
+  Spec 是 MultiClusterService 所需的状态。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/networking-resources/multi-cluster-service-v1alpha1.md:35]: ditto

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/networking-resources/multi-cluster-service-v1alpha1.md [hunk 3]
```diff
@@ -30,153 +30,155 @@ MultiClusterService is a named abstraction of multi-cluster software service. Th
...
+          
+          - 应在此文件中指定内置错误码，并且错误码应使用驼峰法命名。
+             
+          - 特定于云驱动的错误码名称必须符合 foo.example.com/CamelCase 格式。
+             
 
 ## MultiClusterServiceSpec 
 
-MultiClusterServiceSpec is the desired state of the MultiClusterService.
+MultiClusterServiceSpec 是 MultiClusterService 所需的状态。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/networking-resources/multi-cluster-service-v1alpha1.md:143]: ditto

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/networking-resources/multi-cluster-ingress-v1alpha1.md (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/networking-resources/multi-cluster-ingress-v1alpha1.md]: Thanks, updated.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**